core/amber/src/main/scala/edu/uci/ics/texera/workflow/operators/udf/pythonV2/source/RedditSearchSourceOpDesc.java

```diff
@@ -0,0 +1,161 @@
+package edu.uci.ics.texera.workflow.operators.udf.pythonV2.source;
+
+import com.fasterxml.jackson.annotation.JsonProperty;
+import com.fasterxml.jackson.annotation.JsonPropertyDescription;
+import com.kjetland.jackson.jsonSchema.annotations.JsonSchemaTitle;
+import edu.uci.ics.texera.workflow.common.metadata.InputPort;
+import edu.uci.ics.texera.workflow.common.metadata.OperatorGroupConstants;
+import edu.uci.ics.texera.workflow.common.metadata.OperatorInfo;
+import edu.uci.ics.texera.workflow.common.metadata.OutputPort;
+import edu.uci.ics.texera.workflow.common.operators.source.PythonSourceOperatorDescriptor;
+import edu.uci.ics.texera.workflow.common.tuple.schema.Attribute;
+import edu.uci.ics.texera.workflow.common.tuple.schema.AttributeType;
+import edu.uci.ics.texera.workflow.common.tuple.schema.Schema;
+
+import static java.util.Collections.singletonList;
+import static scala.collection.JavaConverters.asScalaBuffer;
+
+
+public class RedditSearchSourceOpDesc extends PythonSourceOperatorDescriptor {
+
+    @JsonProperty(required = true)
+    @JsonSchemaTitle("Client Id")
+    @JsonPropertyDescription("Client id that uses to access Reddit API")
+    public String client_id;
```

Comment:
please use camel case format for variable names, this is java code.

Reply:
solved